splash/tests/test_execute.py

```diff
@@ -2166,8 +2190,8 @@ def test_get_perf_stats(self):
         end
         """
         out = self.request_lua(func).json()
-        self.assertItemsEqual(out.keys(),
-                              ['walltime', 'cputime', 'maxrss'])
+        self.assertEqual(list(out.keys()).sort(),
```

Comment:
`[].sort()` returns None, so this test now checks nothing. It should be `sorted(out.keys())` or `set(out.keys())`.